spark/graphar/src/main/scala/com/alibaba/graphar/importer/Neo4j.scala

```diff
@@ -73,6 +73,9 @@ object Neo4j {
         neo4j.password
       )
       .config("spark.master", "local")
+      .config("spark.sql.legacy.parquet.nanosAsLong", "false")
```

Comment:
these configuration should not set by user. it's better to move to https://github.com/alibaba/GraphAr/blob/bb331dbb96e83089cac941f1eb0f51aedc72d89c/spark/graphar/src/main/scala/com/alibaba/graphar/util/FileSystem.scala#L59

Reply:
Done